frontend/src/metabase/components/UserPicker/UserPicker.jsx

```diff
@@ -24,15 +24,19 @@ const UserPicker = ({ value, validateValue, users, canAddItems, onChange }) => {
     : null;
 
   const options = useMemo(() => {
-    return users.map(user => ({ label: user.common_name, value: user }));
+    // `label` here isn't really used because we specify `filterOption`.
+    // Normally, `options` will be filtered by its `label` if we don't provide
+    // `filterOption` to <TokenField />.
+    return users.map(user => ({ value: user }));
```

Comment:
`label` here isn't really used because we specify `filterOption`. Normally, `options` will be filtered by its `label` if we don't provide `filterOption` to <TokenField />.

https://github.com/metabase/metabase/blob/d0ad25058d96436ee1995e43de4fbdc00c830ca5/frontend/src/metabase/components/TokenField/TokenField.tsx#L206-L209